### ISSUE TITLE
Add Resident populations

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#246]
The app will contain the list and the data of existing experiments to date (about fifteen) and the user will be allowed to toggle on/off these neuronal locations in every new experiment. These 'resident populations' can be visualised in any new experiments. I see this probably taking up some time as it might also need to change the design of the FE to accomodate for the list of the resident populations. @stephenlenzi thoughts? 

point 5a here: https://docs.google.com/document/d/1iAqMKX8dr7YVa1dsDFFzxOl2Wd89fngi/edit#

**Comment by zsinnema** [comment]
@neuroBazzi correct, we need a design for that.

question: can "any uploaded population" become a "resident" population?
For example a user uploads a population (the user is the admin user), then if this is an interesting population the admin user switches on/off (in the admin interface) that this will be a "resident" population

in terms of development hours for the backend this is not a lot (max. 8 including changing the API) but it will give the admin a lot of flexibility

frontend development my best guess is 16 h

**Comment by neuroBazzi** [comment]
@stephenlenzi correct me if I am wrong, but what I understood from the client is that any (i.e. all) populations are resident and can we toggled on/off. I recalled I asked this question during the meeting but I don't recall the answer.

**Comment by zsinnema** [comment]
but that could end up in hundreds of populations to toggle
or do you mean the current set (btw there are no 15 in dev)

**Comment by neuroBazzi** [comment]
Yes there could be many. but let's wait to @stephenlenzi to weigh in on this

**Comment by stephenlenzi** [comment]
Yes this sounds like they want an arbitrary number of experiments available to display. I think (but let's clarify with them) this only applies to the genetic marker example populations that they have provided (i.e. allow all users to view their particular experiment cell type distributions. I think it has the potential to increase in number, but would have some sort of upper limit on it, e.g how many mouse lines exist (probably <100).  The latter just depends on how many experiments they do - its possible they want this, but I think its less likely.

**Comment by stephenlenzi** [comment]
I also think (@neuroBazzi do you remember?) that they would want some sort of sampling from the multiple samples that make up a particular genetic population (i.e. for a given marker, say V1, taking some subset from cords 1,2and 3 and displaying one population that represents the marker V1)

**Comment by afonsobspinto** [comment]
Could you point me to the V1 to V4 populations? (You can either point me to the input or output files of cordmap)

As a summary for the current issue:
- There will be V1 to V4 resident populations for now and there's no option for the users of the application to add a new resident population.
- The resident populations will be read-only (equivalent to not being the owner) for all users. Practically they won't be able to edit the color of the populations
